src/rush.ts

```diff
@@ -0,0 +1,840 @@
+        },
+        ...PROJECT_SELECTION_OPTIONS,
+        {
+          name: ["-v", "--verbose"],
+          description:
+            "Display the logs during the build, rather than just displaying the build status summary",
+        },
+        {
+          name: ["-c", "--changed-projects-only"],
+          description:
+            'Normally the incremental build logic will rebuild changed projects as well as any projects that directly or indirectly depend on a changed project. Specify "--changed-projects-only" to ignore dependent projects, only rebuilding those projects whose files were changed. Note that this parameter is "unsafe"; it is up to the developer to ensure that the ignored projects are okay to ignore',
+        },
+        {
+          name: "--ignore-hooks",
+          description:
+            'Skips execution of the "eventHooks" scripts defined in rush.json. Make sure you know what you are skipping',
+        },
+      ],
+    },
+    {
+      name: "rebuild",
+      description: "Clean and rebuild the entire set of projects",
+      options: [
+        {
+          name: ["-h", "--help"],
+          description: "Show this help message and exit",
+        },
+        {
+          name: ["-p", "--parallelism"],
+          args: { name: "COUNT" },
+          description:
+            'Specifies the maximum number of concurrent processes to launch during a build. The COUNT should be a positive integer or else the word "max" to specify a count that is equal to the number of CPU cores. If this parameter is omitted, then the default value depends on the operating system and number of CPU cores. This parameter may alternatively be specified via the RUSH_PARALLELISM environment variable',
+        },
+        ...PROJECT_SELECTION_OPTIONS,
+        {
+          name: ["-v", "--verbose"],
+          description:
+            "Display the logs during the build, rather than just displaying the build status summary",
+        },
+        {
+          name: "--ignore-hooks",
+          description:
+            'Skips execution of the "eventHooks" scripts defined in rush.json. Make sure you know what you are skipping',
+        },
+      ],
+    },
+    {
+      name: "tab-complete",
+      description: "Provides tab completion",
+      options: [
+        {
+          name: ["-h", "--help"],
+          description: "Show this help message and exit",
+        },
+        {
+          name: "--word",
+          args: { name: "WORD" },
+          description: 'The word to complete. The default value is ""',
+        },
+        {
+          name: "--position",
+          args: { name: "INDEX" },
+          description:
+            "The position in the word to be completed. The default value is 0",
+        },
+      ],
+    },
+  ],
+  options: [
+    {
+      name: ["-h", "--help"],
+      description: "Show this help message and exit",
+    },
```

Comment:
Adding `isPersistent` will add this option to subcommands too. We can remove those.
```suggestion
    {
      name: ["-h", "--help"],
      description: "Show this help message and exit",
      isPersistent: true,
    },
```

Reply:
Thanks for the review. I also removed the redundant help option declaration for subcommands.